xla/service/gpu/runtime/gpublas_lt_matmul.cc

```diff
@@ -109,6 +109,11 @@ absl::Status DoMatmul(
   }));
 
   TF_ASSIGN_OR_RETURN(auto algos, (*plan)->GetAlgorithms());
+  if((size_t)algorithm >= algos.size()) {
```

Comment:
`static_cast<size_t>` or maybe remove it completely?

Reply:
yes, I have added static_cast there, to be sure it also works fine for negative values